app/frontend/src/router/agriSeafoodOpScreening.js

```diff
@@ -1,77 +1,82 @@
 /**
  * Agriculture and Seafood Operator Screening Form Routes
  */
+import { FormNames } from '@/utils/constants';
+
+const baseName = 'AgriSeafoodOpScreening';
+const baseTitle = 'Agriculture and Seafood Operator Screening';
+
 export default [
   {
-    path: '/agriseafoodopscreening',
+    path: `/${FormNames.AGRISEAFOODOPSCREENING}`,
     component: () => import(/* webpackChunkName: "agriseafoodopscreening" */ '@/views/AgriSeafoodOpScreening.vue'),
     children: [
       {
         path: '',
-        name: 'AgriSeafoodOpScreeningForm',
+        name: `${baseName}Form`,
         component: () => import(/* webpackChunkName: "agriseafoodopscreening-form" */ '@/views/agriseafoodopscreening/Root.vue'),
         meta: {
-          title: 'Agriculture and Seafood Operator Screening'
+          title: baseTitle
         }
       },
       {
         path: 'admin',
-        name: 'AgriSeafoodOpScreeningAdmin',
+        name: `${baseName}Admin`,
         component: () => import(/* webpackChunkName: "agriseafoodopscreening-admin" */ '@/views/agriseafoodopscreening/Admin.vue'),
         meta: {
           hasLogin: true,
           requiresAuth: true,
-          title: 'Agriculture and Seafood Admin'
+          title: `${baseTitle} Admin`
         }
       },
       {
         path: 'admin/dashboard',
-        name: 'AgriSeafoodOpScreeningDashboards',
+        name: `${baseName}Dashboards`,
         component: () => import(/* webpackChunkName: "agriseafoodopscreening-dashboard" */ '@/views/agriseafoodopscreening/Dashboards.vue'),
         meta: {
           hasLogin: true,
           requiresAuth: true,
-          title: 'Agriculture and Seafood Admin'
+          title: `${baseTitle} Admin`
```

Comment:
Should this be Dashboards?

Reply:
It's Admin on the mines one too, though that might be a copy paste typo too

Reply:
I don't think it was a copy-paste error - we had it originally as just an admin panel - easy to change but was refactoring this to maintain existing title consistency with prod.